relm-examples/Cargo.toml

```
@@ -29,3 +29,7 @@ version = "^0.24.0"
 [dev-dependencies.relm-test]
 path = "../relm-test"
 version = "^0.24.0"
+
+[dependencies.adw]
```

It should probably be:
```suggestion
[dev-dependencies.adw]
```

---

Honestly, I don't know the difference 😅